### PR TITLE
[DDW-798] "1 confirmations" is displayed on Transaction Assurance Level

### PR DESCRIPTION
**Pull request by DeeJayElly**
This PR fixes the small issue with message display for amount of confirmations. When there is only 1 transaction confirmation it will display "1 confirmation" instead of "1 confirmations" as it was until now.

## Screenshots

Screenshot 2019-08-12 06 44 17

---

## Testing Checklist

- [Slack QA thread](https://input-output-rnd.slack.com/archives/GGKFXSKC6/p1565610787181300)
- [ ] Test if there is a change in single/plural in word "confirmation" based on the number of confirmations

---

### Basics

- [ ] PR has been assigned and has appropriate labels (`feature`/`bug`/`chore`, `release-x.x.x`)
- [ ] PR is updated to the most recent version of the target branch (and there are no conflicts)
- [ ] PR has a good description that summarizes all changes and shows some screenshots or animated GIFs of important UI changes
- [ ] CHANGELOG entry has been added to the top of the appropriate section (*Features*, *Fixes*, *Chores*) and is linked to the correct PR on GitHub
- [ ] Automated tests: All acceptance and unit tests are passing (`yarn test`)
- [ ] Manual tests (minimum tests should cover newly added feature/fix): App works correctly in *development* build (`yarn dev`)
- [ ] Manual tests (minimum tests should cover newly added feature/fix): App works correctly in *production* build (`yarn package` / CI builds)
- [ ] There are no *flow* errors or warnings (`yarn flow:test`)
- [ ] There are no *lint* errors or warnings (`yarn lint`)
- [ ] There are no *prettier* errors or warnings (`yarn prettier:check`)
- [ ] There are no missing translations (running `yarn manage:translations` produces no changes)
- [ ] Text changes are proofread and approved (Jane Wild / Amy Reeve)
- [ ] Japanese text changes are proofread and approved (Junko Oda)
- [ ] UI changes look good in all themes (Alexander Rukin)
- [ ] Storybook works and no stories are broken (`yarn storybook`)
- [ ] In case of dependency changes `yarn.lock` file is updated

### Code Quality
- [ ] Important parts of the code are properly commented and documented
- [ ] Code is properly typed with flow
- [ ] React components are split-up enough to avoid unnecessary re-renderings
- [ ] Any code that only works in main process is neatly separated from components

### Testing
- [ ] New feature/change is covered by acceptance tests
- [ ] New feature/change is manually tested and approved by QA team
- [ ] All existing acceptance tests are still up-to-date
- [ ] New feature/change is covered by Daedalus Testing scenario
- [ ] All existing Daedalus Testing scenarios are still up-to-date

### After Review
- [ ] Merge the PR
- [ ] Delete the source branch
- [ ] Move the ticket to `done` column on the YouTrack board
- [ ] Update Slack QA thread by marking it with a green checkmark

### CODE REVIEW

**Comment by nikolaglumac** [comment]
@DeeJayElly also, injecting store into a component is not something we usually do...

**Comment by DeeJayElly** [comment]
> @DeeJayElly also, injecting store into a component is not something we usually do...

Reverted

**Comment by DeeJayElly** [comment]
> @DeeJayElly I don't think this is a good approach. Not all languages have the same rules.
> Can you please try to see if solution like this (or smth similar) would work? https://github.com/formatjs/react-intl
> Thanks!

Implemented

File: source/renderer/app/i18n/locales/ja-JP.json [hunk 1]
```diff
@@ -571,7 +571,7 @@
   "wallet.transaction.assuranceLevel.high": "高",
   "wallet.transaction.assuranceLevel.low": "低",
   "wallet.transaction.assuranceLevel.medium": "中",
-  "wallet.transaction.confirmations": "{confirmationsNumber, plural, one {# 確認} =21 {#+ 確認} other {# 確認}}",
+  "wallet.transaction.confirmations": "{confirmationsNumber, plural, one {# 確認} =21 {20+ 確認} other {# 確認}}",
```
Comment (nikolaglumac) [inline source/renderer/app/i18n/locales/ja-JP.json]: @DeeJayElly are we sure the Japanese plural is the same as singular?

Comment (DeeJayElly) [reply source/renderer/app/i18n/locales/ja-JP.json]: Yes, its confirmed.

Screenshot 2019-08-13 03 51 54

Comment (nikolaglumac) [reply source/renderer/app/i18n/locales/ja-JP.json]: Perfect! Thanks!

**Comment by IuliaDolishniak** [comment]
English looks good, I am also concerned on the Japanese translation as it doesn't change accordingly.

**Comment by nikolaglumac** [comment]
@IuliaDolishniak Japanese is correct: https://github.com/input-output-hk/daedalus/pull/1531#discussion_r313284825

**Pull request merged**